diff --git a/kernel/config-arm64 b/kernel/config-arm64
index 24e820a9..b0e4a00a 100644
--- a/kernel/config-arm64
+++ b/kernel/config-arm64
@@ -3503,7 +3503,15 @@ CONFIG_RPCSEC_GSS_KRB5=y
 # CONFIG_SUNRPC_DISABLE_INSECURE_ENCTYPES is not set
 # CONFIG_SUNRPC_DEBUG is not set
 # CONFIG_CEPH_FS is not set
-# CONFIG_CIFS is not set
+CONFIG_CIFS=y
+# CONFIG_CIFS_STATS2 is not set
+CONFIG_CIFS_ALLOW_INSECURE_LEGACY=y
+CONFIG_CIFS_UPCALL=y
+CONFIG_CIFS_XATTR=y
+CONFIG_CIFS_POSIX=y
+# CONFIG_CIFS_DEBUG is not set
+CONFIG_CIFS_DFS_UPCALL=y
+# CONFIG_CIFS_SMB_DIRECT is not set
 # CONFIG_SMB_SERVER is not set
 # CONFIG_CODA_FS is not set
 # CONFIG_AFS_FS is not set
